index.js

```diff
@@ -91,6 +91,20 @@ function findFocusElement(details, dialog): ?HTMLElement {
   }
 }
 
+function toggleDetails(details: Element, open: boolean) {
+  // Don't update unless state is changing
+  if (open === details.hasAttribute('open')) return
+
+  const summary = details.querySelector('summary')
+  if (summary) {
+    // Toggle via clicking summary so it can be canceled by listeners wanting
+    // to prevent the toggle
+    summary.click()
+  } else {
+    open ? details.setAttribute('open', 'open') : details.removeAttribute('open')
```

Comment:
The `open` attribute is a [boolean attribute](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-details-element) so it's value should be an empty string.

Reply:
Sounds good, just want to mention that this PR did not change this behavior, it was previously setting the value to `'open'`.

https://github.com/github/details-dialog-element/pull/21/files#diff-168726dbe96b3ce427e7fedce31bb0bcL150